blink.js

```diff
@@ -1 +1,11 @@
-// YOUR CODE GOES HERE
+(function($) {
+    $.fn.blink = function(time) {
+        this.each(function() {
+            var elem = $(this);            
```

Comment:
Minor, but I would use the convention of prefixing jQuery object variable/property names with `$` (i.e. using `$elem` here).